moveit_planners/chomp/chomp_motion_planner/src/chomp_trajectory.cpp

```diff
@@ -170,6 +183,44 @@ void ChompTrajectory::updateFromGroupTrajectory(const ChompTrajectory& group_tra
   }
 }
 
+
+void ChompTrajectory::fillInLinearInterpolation()
+{
+  double start_index  = start_index_ - 1;
+  double end_index    = end_index_ + 1;
+  int time_steps = end_index - start_index;
+  for (int i = 0; i < num_joints_; i++)
+  {
+    double theta = ((*this)(end_index, i) - (*this)(start_index,i)) / (end_index- 1);
+    for(int j=start_index+1 ; j< end_index ; j++)
```

Comment:
did you run clang-format? there should be spaces around ``=``

Reply:
yes I run clang-format in my latest commit.

Reply:
Travis disagrees:
https://travis-ci.org/ros-planning/moveit/jobs/398829070

Reply:
travis (all tests) pass currently.